src/pdmat.jl

```diff
@@ -30,13 +56,11 @@ Base.propertynames(::PDMat) = (:mat, :chol, :dim)
 AbstractPDMat(A::Cholesky) = PDMat(A)
 
 ### Conversion
+Base.convert(::Type{PDMat{T,S}}, a::PDMat{T,S}) where {T<:Real,S<:AbstractMatrix{T}} = a
```

Comment:
Is the convention to define this no-op `convert` method? Or should one rather define
```julia
PDMat{T,S}(pdm::PDMat{T,S}) where {T,S} = pdm
```
?

Reply:
I like your suggestion, implemented.